bootstrap.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# We could do something much fancier here.
+
+# Fetch dart dependencies
+cd packages/shorebird_cli
+dart pub get
+cd ../..
+
+cd packages/shorebird_code_push_api
+dart pub get
+cd ../..
+
+cd packages/shorebird_code_push_api_client
+dart pub get
+cd ../..
+
+cd updater/dart_bindings
+dart pub get
+cd ../..
+
+cd updater/dart_cli
+dart pub get
+cd ../..
```

Comment:
FYI

`very_good_cli` supports a recursive packages get so this could theoretically all be replaced with:

`very_good packages get -r ./packages`

Reply:
Happy to try it. 🤷

Reply:
Eventually we have to offer a solution for this problem to our customers (I think?).  Whether that's vending vg_cli or melos or whatever I don't know.